plugins/modules/gitlab_group.py

```diff
@@ -205,11 +205,15 @@ def create_or_update_group(self, name, parent, options):
                 'name': name,
                 'path': options['path'],
                 'parent_id': parent_id,
-                'visibility': options['visibility'],
-                'project_creation_level': options['project_creation_level'],
-                'auto_devops_enabled': options['auto_devops_enabled'],
-                'subgroup_creation_level': options['subgroup_creation_level'],
             }
+            if options.get('auto_devops_enabled'):
```

Comment:
This is a boolean.
```suggestion
            if options.get('auto_devops_enabled') is not None:
```

Reply:
Since there were already some `if` based checks and I wanted to restore the module to a working state, I figured I keep the changes as minimal as possible. As @nejch also comments to check for `None` values, want me to add it to `create_group` function instead?